plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -140,7 +230,7 @@ private void createPromoteSection(Composite parent) {
     autoPromoteButton = new Button(promoteComp, SWT.CHECK);
     autoPromoteButton.setText(Messages.getString("auto.promote"));
     autoPromoteButton.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, true, false));
-    
+
     Link manualPromoteLabel = new Link(promoteComp, SWT.NONE);
```

Comment:
It may be worth extracting the selection listener and turn it into a utility class in .ui.util that opens a browser on the `SelectionEvent`'s `text` field (the value of the `href` attribute).